base/dataset_test.go

```diff
@@ -88,9 +91,20 @@ func TestRawDatasetRefs(t *testing.T) {
 
 	ctx := context.Background()
 	r := newTestRepo(t)
-	addCitiesDataset(t, r)
+	s, err := collection.NewLocalSet(ctx, event.NilBus, "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	ref := addCitiesDataset(t, r)
```

Comment:
This comes up oftern and feels a bit unergonomic, maybe we can push this whole chunk into the addCitiesDataset given we can push the collection into repo or get rid of repo.